database.sql

```diff
@@ -1,6 +1,6 @@
 -- ------------------------------------------
 -- Friendica 2023.09-dev (Giant Rhubarb)
--- DB_UPDATE_VERSION 1518
+-- DB_UPDATE_VERSION 1519
```

Comment:
Remember to increase the database version number.

Reply:
Of course you could have also merged this PR before yours!